### PR TITLE
openssl@3 3.2.0

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/o/openssl@3.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.1/openssl-3.1.4.tar.gz"
-  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.1.4.tar.gz"
-  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.1/openssl-3.1.4.tar.gz"
-  sha256 "840af5366ab9b522bde525826be3ef0fb0af81c6a9ebd84caa600fea1731eee3"
+  url "https://www.openssl.org/source/openssl-3.2.0.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.2.0.tar.gz"
+  mirror "https://www.openssl.org/source/old/3.1/openssl-3.2.0.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.1/openssl-3.2.0.tar.gz"
+  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.2.0.tar.gz"
+  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.1/openssl-3.2.0.tar.gz"
```
Comment (Bo98) [inline Formula/o/openssl@3.rb:9]:
```suggestion
  mirror "https://www.openssl.org/source/old/3.2/openssl-3.2.0.tar.gz"
  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.2/openssl-3.2.0.tar.gz"
  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.2.0.tar.gz"
  mirror "http://www.mirrorservice.org/sites/ftp.openssl.org/source/old/3.2/openssl-3.2.0.tar.gz"
```

**Review by Bo98 — COMMENTED** [review]
Maybe worth updating the Perl build deps.

File: Formula/o/openssl@3.rb (hunk 1 above)
Comment (p-linnane) [reply Formula/o/openssl@3.rb]: Fixed and resources updated. Thanks for the reminder.

**Review by chenrui333 — APPROVED** [review]

**Review by ZhongRuoyu — APPROVED** [review]

**Comment by chenrui333** [comment]
```
brew test --retry --verbose ctpv
brew test --retry --verbose gatsby-cli
brew test --retry --verbose glslviewer
brew test --retry --verbose subversion
```
glslviewer test got updated.

I cannot reproduce the test issues with gatsby-cli and subversion

**Comment by chenrui333** [comment]
subversion test failures on 14 and 14-arm
rpm test failure on linux build

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**